diff --git a/.vscode/settings.json b/.vscode/settings.json
new file mode 100644
index 0000000000000000000000000000000000000000..f6e1d59217c5a9a7b520c5b78336c519ca2360f1
--- /dev/null
+++ b/.vscode/settings.json
@@ -0,0 +1,6 @@
+{
+    "eslint.validate": [
+        "glimmer-ts",
+        "glimmer-js"
+    ]
+}
\ No newline at end of file
diff --git a/addon/models/module.js b/addon/models/module.js
index 4887ba3bd30829653cd6944f1097ed65db41a9da..8c10bc597ef794f99e52549d0bef31eae12b08a1 100644
--- a/addon/models/module.js
+++ b/addon/models/module.js
@@ -13,7 +13,7 @@ export default class Module extends Model {
   @hasMany('class', { async: false, inverse: null })
   classes;

-  @hasMany('class', { async: false, inverse: null })
+  @hasMany('component', { async: false, inverse: null })
   components;

   /*
